### PR TITLE
add Kong and Tyk to API Gateway OSS Alternatives

### PR DESCRIPTION
**Pull request by thibaultcha**
Disclaimer: I am one of the maintainers of [Kong](https://github.com/Mashape/kong).

Just proposing some of the popular OSS solutions to APIs and Microservices management. Let me know if you need more context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1087,6 +1087,11 @@ API Gateway
 -	It allows “serverless” deployment of applications built with Lambda.
 -	🔸Switching over deployments after upgrades can be tricky. There are no built-in mechanisms to have a single domain name migrate from one API gateway to another one. So it may be necessary to build an additional layer in front (even another API Gateway) to allow smooth migration from one deployment to another.
 
+### API Gateway Alternatives and Lock-In
+
+- [Kong](https://getkong.org): Open-source, on-premises API and Microservices Gateway built on nginx with Lua. Kong is extensible through "plugins".
```
Comment (jlevy) [inline README.md]: Nits: capitalization and Unicode quotes

> - [Kong](https://getkong.org) is an open-source, on-premises API and microservices gateway built on nginx with Lua. Kong is extensible through “plugins”.
>- [Tyk](https://tyk.io) is an open-source API gateway implemented in Go and available in the cloud, on-premises or hybrid.

**Comment by jlevy** [comment]
@ehammond @ThanosBaskous thoughts?

**Comment by ehammond** [comment]
@jlevy I have few opinions on alternatives to AWS services. I prefer to use the AWS ecosystem and take advantage of the powers of cross-service integrations that are constantly being added. Not saying that other alternatives aren't good, just that I don't spend much time trying them out.

**Comment by thibaultcha** [comment]
Patch updated with @jlevy 's text suggestions.

**Comment by ThanosBaskous** [comment]
Thank you for your contribution, @thibaultcha!

**Pull request merged**